src/Sulu/Component/Cache/ConfigCache.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Cache;
+
+use Symfony\Component\Config\ConfigCache as BaseConfigCache;
+
+class ConfigCache extends BaseConfigCache
```

Comment:
when i use the normal `ConfigCache` i cannot get the content from that class. then i have to inject the `$cache` variable and the path to the cache file ... but i will add a phpdoc.